index.js

```diff
@@ -66,8 +66,9 @@ function client(apiKey, config) {
 
   return {
     accounts: resources.accounts(requestOptions, maybePromiseRequest),
+    customers: resources.customers(requestOptions, maybePromiseRequest),
     merchants: resources.merchants(requestOptions, maybePromiseRequest),
-    orders: resources.orders(requestOptions, maybePromiseRequest)
+    orders: resources.orders(requestOptions, maybePromiseRequest),
```

Comment:
nit: this repo is currently formatted without trailing commas at the end of objects.  It _is_ Button style now, but let's keep it consistent here and upgrade all instances at once in a separate change if we want.